package.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "quiz",
+    "description": "Quiz",
+    "version": "1.0.0",
+    "scripts": {
+        "build": "webpack --env.mode=prod --env.watch=false",
+        "start": "webpack --env.mode=dev --env.watch=true",
+        "zip": "cd ./output && bestzip ./ActionPackage.zip *",
+        "create": "npm run build && npm run zip && upload-action-package -z ./output/ActionPackage.zip -t ./output/ -e https://int.actions.office365.com",
+        "update": "npm run build && npm run zip && upload-action-package -z ./output/ActionPackage.zip -e https://int.actions.office365.com",
+        "inner-loop": "create-inner-loop -p com.microsoft.sample.quiz.mike06072009 -d ./output -e https://int.actions.office365.com",
+        "delete-loop": "delete-inner-loop -p com.microsoft.sample.quiz.mike06072009 -e https://int.actions.office365.com"
+    },
+    "devDependencies": {
+        "@types/react": "16.8.19",
+        "@types/react-dom": "16.8.0",
+        "action-package-deploy": "^1.0.23",
+        "bestzip": "^2.1.6",
+        "clean-webpack-plugin": "^3.0.0",
+        "copy-webpack-plugin": "5.0.3",
+        "crossorigin-webpack-plugin": "0.1.4",
+        "file-loader": "^6.2.0",
+        "html-loader": "^1.3.2",
+        "html-webpack-plugin": "3.2.0",
+        "node-sass": "^4.14.1",
+        "sass-loader": "7.1.0",
+        "ts-loader": "5.4.5",
+        "typescript": "3.5.3",
+        "webpack": "4.37.0",
+        "webpack-cli": "^3.3.12"
+    },
+    "dependencies": {
+        "@fluentui/react-northstar": "^0.51.3",
```

Comment:
"@fluentui/react-northstar" Not required


Also please remove the other unused dependencies